katas/content/superposition/even_odd/solution.md

```diff
@@ -0,0 +1,44 @@
+### Solution
+
+Let’s look at some examples of basis states to illustrate the binary numbering system. 
+
+The 4 basis states on $N = 2$ qubits can be split in two columns, where the left column represents the basis states that form the required superposition state for `isEven = true` and the right column - the basis states that form the required superposition state for `isEven = false`.
+
+| even | odd |
+| ---- | --- |
+| 00   | 01  |
```

Comment:
Nice tables! I would make first digits bold in the first table, so that the format matches the rest of the tables